### PR TITLE
Update linux apache configuration to refer to fcgi

### PR DESCRIPTION
**Pull request by liayn**

### CODE REVIEW

**Comment by shoeper** [comment]
I'm not actually sure if we should replace all the old documentation. It is for Apache 2.2 according to what I know and a well working setup.

@freeplant What do you think? I think having a good apache 2.4 entry in the manual is important, so we should somehow get this in the manual.

**Comment by liayn** [comment]
I'm open for all suggestions, no problem. I just consider Apache 2.2 extremely outdated and no serious webhoster would ever use this version nowadays.

**Comment by shoeper** [comment]
Let's wait for Daniels decision.

**Comment by freeplant** [comment]
Let's replace the manual from Apache v2.4 to Apache v2.2. @shoeper Can you give it a test? (testing root-domain part is enough)

File: deploy/deploy_with_apache.md [hunk 1]
```diff
@@ -4,36 +4,20 @@
...
+
 ## Prepare
 
-`Notice`: Seahub has an issue working with Apache 2.4 using the mod_fastcgi https://github.com/haiwen/seafile/issues/1258.
+Install and enable apache modules
 
-1. Install and enable mod_fastcgi and also enable mod_rewrite. On Ubuntu:
+	On Ubuntu you can use:
 
     ```bash
```
Comment (shoeper) [inline deploy/deploy_with_apache.md]: This code block isn't rendered correctly for some reason.

File: deploy/deploy_with_apache.md [hunk 2]
```diff
@@ -181,20 +120,18 @@ When a user visit https://example.com/home/my/, Apache receives this request and
...
-```
+    SetEnvIf Request_URI . proxy-fcgi-pathinfo=unescape
+    SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
+    ProxyPass / fcgi://127.0.0.1:8000/
 
 When a user click a file download link in Seahub, Seahub reads the value of `FILE_SERVER_ROOT` and redirects the user to address `https://example.com/seafhttp/xxxxx/`. `https://example.com/seafhttp` is the value of `FILE_SERVER_ROOT`. Here, the `FILE_SERVER` means the FileServer component of Seafile, which only serves for raw file downloading/uploading.
 
 When Apache receives the request at 'https://example.com/seafhttp/xxxxx/', it proxies the request to FileServer, which is listening at 127.0.0.1:8082. This is controlled by the following config items:
 
 ```apache
```
Comment (shoeper) [inline deploy/deploy_with_apache.md]: Here's also some problem with the codeblock. Looks like the previous one hasn't been closed

**Comment by shoeper** [comment]
> Let's replace the manual from Apache v2.4 to Apache v2.2

I think you mean it the other way around ;)

I've had a look on it and the commit seems to be good to merge - but I'm no Apache expert. Testing an apache setup in deph would take me at least one hour. I'd have to settle it with Alexander. Should I do it?

I'm quite sure that this is working for @liayn and I don't see reasons why it shouldn't work in general.

**Comment by liayn** [comment]
@shoeper Will take a look at the rendering issues. And, yes, this is 1:1 the live-setup we are running for nearly 2 years now.

**Comment by shoeper** [comment]
Looks good.

**Comment by shoeper** [comment]
@freeplant

**Comment by liayn** [comment]
Thanks.

**Pull request merged**